core/templates/dev/head/components/RatingComputationService.js

```diff
@@ -46,6 +46,14 @@ oppia.factory('RatingComputationService', [function() {
 
         return totalValue / totalNumber;
       }
+    },
+    getRelativeChangeInAverageRating: function(newRating, oldRating) {
+      if (!oldRating) {
+        return '+' + parseFloat(newRating).toFixed(2);
```

Comment:
It's used to limit the difference to just 2 places after the decimal. If I don't use this, then I get results which have many decimal places!
Yes, both args are numbers